app/src/NativeBase/Assets/Icons/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * @file Exports icons for easy access elsewhere.
+ */
+
+export { default as Home } from './Home.svg'
```

Comment:
I thought they were but worth checking!

Reply:
I checked with Mia and the icons are from iconify. Instead of adding more libraries for each new icon I have started to collect them in a custom library containing a selection of icons we use, starting with these four. Over time we can add the other icons we use to this library which will reduce the number of load time once the design is more final and the app goes into prod. 
The previously added SVGs have been deleted. 
The app may need npm install again.